src/lsptoolshost/languageStatusBar.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { RoslynLanguageServer } from './roslynLanguageServer';
+import { RoslynLanguageServerEvents } from './languageServerEvents';
+import { languageServerOptions } from '../shared/options';
+import { ServerState } from './serverStateChange';
+import { getCSharpDevKit } from '../utils/getCSharpDevKit';
+
+export function registerLanguageStatusItems(
```

Comment:
Not relevant for this PR, but something to check when we do TFMs - can two different extensions register language status items for the same language?  Devkit does have a language status item (launch.json stuff).